addons/portal/wizard/portal_wizard.py

```diff
@@ -61,7 +61,7 @@ def action_open_wizard(self):
 
     def _action_open_modal(self):
         """Allow to keep the wizard modal open after executing the action."""
-        self.refresh()
+        self.invalidate_cache()
```

Comment:
I think this invalidation should be removed completely.
As far as I can see, it only is a cache inconsistency bomb waiting to be put on fire.